Scripts/Services/FireCasino/DiceGames.cs

```diff
@@ -126,12 +126,6 @@ public virtual void Reset()
             BettingOn = -1;
             Stage = GameStage.Betting;
 
-            if (Roll != null)
```

Comment:
Is this going to cause an issue?

Reply:
It's fine, Roll is always nullified before being checked for not-null, meaning the if-statement was not necessary.

Reply:
Haha, I should know, I wrote that!  It's been a while though.  Good catch.

Reply:
Once i realized how many of these unneeded null checks there were(100+) i stopped removing them, none are in hot paths so shouldn't be a performance issue leaving them.